src/css/components/actions.scss

```diff
@@ -6,7 +6,7 @@
 
 .action-link {
   color: $color-blue;
-  font-size: $sans-s3;
+  font-size: inherit;
```

Comment:
Set the font size of action links to inherit font size and not set explicitly. This allows for more flexibility.